### PR TITLE
Fixes unnecessary emitter fires

### PR DESCRIPTION
**Pull request by silicons**
why....

### CODE REVIEW

**Comment by Cyberboss** [comment]
That's a good question, you should find out instead of bandaiding the issue

**Comment by kevinz000** [comment]
@Cyberboss I've used sdql to try to select nullspace emitters but it didn't show any

**Comment by Cyberboss** [comment]
Then this wouldn't fix it either, would it?

File: code/modules/power/singularity/emitter.dm [hunk 1]
```diff
@@ -185,12 +185,14 @@
 		fire_beam()
 
 /obj/machinery/power/emitter/proc/fire_beam(mob/user)
+	if(!isloc(loc))			//One was somehow processing in nullspace or something.
```
Comment (optimumtact) [inline code/modules/power/singularity/emitter.dm]: this is not a fix, it's covering up a bug

you know better, remove this

**Review by optimumtact — CHANGES_REQUESTED** [review]

**Comment by optimumtact** [comment]
in fact just no, this is terrible

**Pull request closed without merging**